src/main/java/src/chapter5/question4/Question4.java

```diff
@@ -0,0 +1,19 @@
+package src.chapter5.question4;
+
+public class Question4 {
+    public static String concatStringsOfArray(String[] strArray) {
+
+        if (strArray == null) {
+            return null;
+        }
+        var currentString = strArray[0];
+        for (int i = 1; i < strArray.length; i++) {
```

Comment:
This does not handle correctly cases where the string array has a null on the first position and strings on the rest.

Reply:
Fixed.